performance_manager/lib/postgres_utils.py

```diff
@@ -46,7 +46,10 @@ def get_experimental_engine(
     database to be stood up. great for testing from within the shell.
     """
     engine = sa.create_engine(
-        "sqlite+pysqlite:///:memory:", echo=echo, future=True
+        "sqlite+pysqlite:///./test.db",
```

Comment:
does this create the db file? or do you have to initiate it first?

Reply:
Ah yes, that just creates the db file on the local disk. I did that for debugging because the db in memory doesn't allow other applications to attach.